talk/owt/sdk/include/objc/OWT/OWTRemoteMixedStream.h

```diff
@@ -10,6 +10,7 @@ RTC_EXPORT
   @brief Triggers when video layout is changed.
 */
 - (void)streamDidChangeVideoLayout:(OWTRemoteMixedStream*)stream;
+- (void)streamDidChangeActiveInput:(NSString*)activeAudioInputStreamId;
```

Comment:
Please add API description for newly added public APIs.

Reply:
Changes done